__tests__/app.test.js

```diff
@@ -1,8 +1,47 @@
 const request = require('supertest');
 const db = require('../db/connection');
 const seed = require('../db/seeds/seed');
-const testTopicData = require('../db/data/test-data/topics');
+const {articleData, commentData, topicData, userData} = require('../db/data/test-data/index');
 const { app } = require("../app");
 
 afterAll(() => db.end());
-beforeEach( () => seed(testTopicData));
\ No newline at end of file
+beforeEach( () => seed({topicData, userData, articleData, commentData}));
+
+describe('1. GET api topics', () => {
+    describe('a. status 200 & Data', () => {
+        it('sends a 200 request back to the user', () => {
+            return request(app)
+                .get('/api/topic')
+                .expect(200)
+        })
+        it('returns an array back to the user', () => {
+            return request(app)
+                .get('/api/topic')
+                .expect(200)
+                .then( (res) => {
+                    const topics = res.body;
+                    expect(topics).toBeInstanceOf(Array);

```

Comment:
it's good to have this line checking for the data to be an array, you can make it more precise by checking the length of the array matches the number of topics in the test data instead if you wish